lib/public/Settings/IManager.php

```diff
@@ -34,50 +35,64 @@
 interface IManager {
 	/**
 	 * @since 9.1.0
+	 * @depreacted 29.0.0 Use SETTINGS_ADMIN instead
```

Comment:
```suggestion
	 * @depreacted 29.0.0 Use {@see self::SETTINGS_ADMIN} instead
```
Makes it clickable in IDEs